searx/plugins/infinite_scroll.py

```diff
@@ -0,0 +1,8 @@
+from flask_babel import gettext
+
+name = gettext('Infinite scroll')
+description = gettext('Automatically load next page when scrolling to bottom of current page')
+default_on = True
```

Comment:
I'd leave it `False` by default. Extra files need to be downloaded even for users who cannot use this feature (eg js blocking).